### ISSUE TITLE
CANNOT LINK EXECUTABLE "iptables": cannot locate symbol "posix_spawn" referenced by "/data/data/com.termux/files/usr/lib/libxtables.so"

### ISSUE THREAD

**Issue by desperado86** [termux/termux-packages#14623]
### Problem description

iptables is already the newest version (1.8.8).
when i run  iptables, then it error
1.
~ $ iptables
CANNOT LINK EXECUTABLE "iptables": cannot locate symbol "posix_spawn" referenced by "/data/data/com.termux/files/usr/lib/libxtables.so"...
Aborted
2.
~ $ termux-chroot
~ $ iptables
CANNOT LINK EXECUTABLE "iptables": cannot locate symbol "posix_spawn" referenced by "/usr/lib/libxtables.so"...
Aborted

### What steps will reproduce the bug?

now , the  termux can't run iptables . 
thanks a lot , expected your answer.

### System information

termux-info:

```
Termux Variables:
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.sahilister.in/termux/termux-main stable main
Updatable packages:
All packages up to date
termux-tools version:
1.35.0
Android version:
8.0.0
Kernel build information:
Linux localhost 3.18.71-perf-g5edcf86 #1 SMP PREEMPT Wed Oct 23 18:37:20 CST 2019 aarch64 Android
Device manufacturer:
Xiaomi
Device model:
MI 5s
```

**Comment by xtkoba** [comment]
This issue is already fixed in `iptables` version `1.8.9` in commit 852bbf039505d5c9f97c3d9a20e5f1ad79156930.

Note that this issue did exist in `iptables` version `1.8.8`, and it manifested itself only on Android 8.x or before.

**Comment by desperado86** [comment]
> [Newest version is 1.8.9](https://github.com/termux/termux-packages/blob/master/root-packages/iptables/build.sh#L5).
> 
> Where does the `iptables` program come from? You don't seem to be subscribed to the root repo, so I guess you are not using our iptables package.
> 
> Termux's iptables package works fine when I test it.



> [Newest version is 1.8.9](https://github.com/termux/termux-packages/blob/master/root-packages/iptables/build.sh#L5).
> 
> Where does the `iptables` program come from? You don't seem to be subscribed to the root repo, so I guess you are not using our iptables package.
> 
> Termux's iptables package works fine when I test it.

thanks a lot  , i upgraded , it's o k.

**Comment by desperado86** [comment]
> This issue is already fixed in `iptables` version `1.8.9` in commit [852bbf0](https://github.com/termux/termux-packages/commit/852bbf039505d5c9f97c3d9a20e5f1ad79156930).
> 
> Note that this issue did exist in `iptables` version `1.8.8`, and it manifested itself only on Android 8.x or before.

thank you , yes ,it's ok .
